### DISCUSSION TITLE
250 2.0.0 Ok: queued as

### DISCUSSION THREAD

**Discussion by SaasX** [postalserver/postal#2756] [Help with using Postal]
Hi all, 

I found on some of my Postal servers, messages are stuck in the waiting list.

They are marked as sent in the interface, but in the details :
```
250 2.0.0 Ok: queued as 76E39505B7
Time: 0.13s
Support Ref: KRZTVMQX
```

Will the emails be sent later or not?
Configuration error?

Thx

**Comment by willpower232** [comment]
`250` means the receiving email server said the message was okay so whatever happens to it now is out of Postals hands.

For example, Microsoft/Outlook always say "queued for delivery" so you don't know if the email made it to the recipient or was caught by a spam filter along the way.

If your recipient has not received the message, they should pass that 76E39505B7 to their email provider for analysis.

**Reply by SaasX** [reply]
Thanks for your return.

So for you, no worries on my side.

It is simply the recipient who applies their antispam filter to decide whether or not to deliver the email.

And in case of error, I will get a bounce message.

I understood well ?

**Reply by Pacerino** [reply]
It depends on the error. If an error occurs when sending the email, the receiving mail server returns an error message with an error code. If the message is received by the receiving mail server, but an error occurs there, e.g. the mailbox is full or the address no longer exists, then you will receive a bounce message on your return path.

Attached you will find 1. a error message from the receiving mail server (The mail never actually reached the mail server) and 2. a bounce message (The mail has arrived, but could not be delivered to the recipient)
![image](https://github.com/postalserver/postal/assets/22194063/8b497116-517b-4a3c-b5a0-b66ce1e6366c)
![image](https://github.com/postalserver/postal/assets/22194063/2f9f050f-a95a-4ad8-9947-f0406ac76f53)

**Reply by SaasX** [reply]
I'm only wondering about the code **_250 2.0.0 Ok: queued as_**


This is something I don't know

**Reply by willpower232** [reply]
@SaasX yes, you understood it :pray: 

there are of course always other errors available as @Pacerino mentioned but if Postal says the message was sent, you've done as much as you can

**Comment by SaasX** [comment]
Thank for your time all ;)
